docs/articles/templates/dependencies.md

```diff
@@ -21,4 +23,13 @@ const getArticleHeaderHTML = ClientFunction(() => articleHeader().innerHTML, {
 
 > When a client function calls a [selector](/testcafe/documentation/reference/test-api/selector/) internally,
 > the selector does not wait for the element to appear in the DOM
-> and is executed at once, like a client function.
+> and is executed at once, like a client function.  
+
+When dependencies are passed to a client function, [TypeScript](https://www.typescriptlang.org/) cannot find them during compilation. This happens because dependencies are added to the function's scope at runtime and can cause an error:
+
+```sh
+Error: TypeScript compilation failed.
+Cannot find name 'dependencyFoo'.
+```
+
+You can mitigate this by using [`// @ts-ignore`](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-6.html#suppress-errors-in-ts-files-using--ts-ignore-comments) TypeScript feature.
```

Comment:
Mitigate is too vague here: does it remove the error? Or it makes it less frequent?

```
Add the [`// @ts-ignore`](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-6.html#suppress-errors-in-ts-files-using--ts-ignore-comments) TypeScript directive (?) to suppress/avoid this error.
```

Reply:
'by using' is the kind of '-ing' that can almost always be avoided.

Reply:
With '@ts-ignore', TypeScript compiler ignores the error. Changed to 

`Add the [`// @ts-ignore`](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-6.html#suppress-errors-in-ts-files-using--ts-ignore-comments) TypeScript comment to suppress this error.`